src/patternfly/components/DatePicker/date-picker.scss

```diff
@@ -1,21 +1,22 @@
 // @debug $date-picker; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
 
-.#{$date-picker} {
+:root, .#{$date-picker} {
```

Comment:
This could be a followup since we don't show examples with an input label like Figma seems to show ("Date of birth" above the datepicker), but we should add a var for spacing between a datepicker text label and the input itself (would also need to add an example).